app/views/schools/assign_existing_mentor_wizard/review_mentor_eligibility.html.erb

```diff
@@ -0,0 +1,17 @@
+<%
+  page_data(
+    title: "#{@wizard.context.mentor_teacher_full_name} can receive mentor training",
+    backlink_href: new_schools_ect_mentorship_path(@wizard.context.ect_at_school_period, preselect: @wizard.mentor_period_id)
+  )
+%>
+
+<p class="govuk-body">Our records show that <%= @wizard.context.mentor_teacher_full_name %> can get up to 20 hours of ECTE mentor training as your school is working with a DfE-funded training provider.</p>
+
+<p class="govuk-body">We’ll pass on their details to <%= @wizard.context.ect_lead_provider&.name %> who will contact them to arrange the training.</p>
+
+<%= form_with(model: @wizard.current_step, url: @wizard.current_step_path) do |f| %>
+  <%= f.govuk_submit "Continue" %>
+<% end %>
+
+<%= govuk_link_to "#{@wizard.context.ect_lead_provider&.name} will not be providing mentor training to #{@wizard.context.mentor_teacher_full_name}",
```

Comment:
needs wrapping a <p> tag so it renders correctly

Reply:
Thanks! Done